noxfile.py

```diff
@@ -78,6 +81,12 @@ def run_integration(session):
     """Run the integration tests."""
     set_environment_variables(PYBAMM_ENV, session=session)
     session.install("setuptools", silent=False)
+    if (
+        "CI" in os.environ
+        and sys.version_info[:2] == (3, 12)
+        and sys.platform == "linux"
+    ):
+        session.install("pytest-github-actions-annotate-failures")
```

Comment:
```suggestion
```

I think you forgot to remove these lines from the integration tests?

Reply:
If we remove these lines, integration test failures wont be annotated since coverage runs just the unit tests.
See : https://github.com/pybamm-team/PyBaMM/blob/48b4e48d776ad2c254e5fae8db94706c57a8cc38/noxfile.py#L150

Reply:
Oops – that slipped my mind. Thanks!